### ISSUE TITLE
Retrospectives: Tablist "Collect/Group/Vote/Act keyboard" missing tablist label, confusing selected state announcement "Selected Not Selected Group workflow stage"

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1319]
#A11y #accessibility issue
**Describe the bug**
Install the Retrospectives Extension:

Go to the Azure DevOps Marketplace and search for the "Retrospectives" extension.
Click "Get it free" and follow the prompts to install it to your Azure DevOps organization.
Create a Retrospective:

Navigate to your Azure DevOps project.
In the left-hand menu, select "Boards" and then "Retrospectives."
Click on "New Retrospective" to create a new session.
Tab to the horizontal tablist, focus on Collect Group Vote and Act
Observe how if the tablist has a label, and how tabs are announced (their name and selected state etc.)
Actual result:
The tablist is missing a label.  Expect the label to be something like "Workflow stage"
The selected state are incorrect and confusing, it is hard to tell whether the tab is selected. For examples
NVDA announces the Collect tab (which is correctly not selected) as "Not selected Collect workflow stage tab selected 1 of 4" 
And for the Group tab which is correct selected, NVDA announces it as "Selected Group workflow state tab selected 2 of 4"
In the aria-label for the tabs, "Selected" and "Not selected" was included in the label.  This should not be the case.  "<div class="retrospective-workflowState" aria-setsize="4" aria-posinset="1" aria-label="Not selected Collect workflow stage" role="tab" tabindex="0"><p class="stage-text">Collect</p></div>"
The tabs' label is very verbose, it can just be "Collect", "Group", "Vote" and "Act".  The tablist label should already provide the context "Workflow stage", no need to repeat.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/04dbaec1-2903-4fc6-bd71-c330a2421a30" />

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/60d24d4b-f3d8-4616-b53e-dc76e5c655b7" />

**Expected behavior**
Tablist should have a label that describe what the tablist is for, for example "Workflow state"
Each tab has a label that contains its visual label, descriptive and concise:  for examples: "Collect", "Group", "Vote" and "Act"
When the item is focused, the selected or non selected state are announced correctly and clearly.
As an example, when the user tab into the list and move through the tabs, screen reader announces something like this: 
Workflow state  tab control
Collect  tab  selected  1 of 4
Group  tab  2 of 4
Vote  tab  3 of 4
Act  tab  4 of 4

**What screen type are you seeing this on ?**
[ ] Desktop
